docs/eks.md

```diff
@@ -0,0 +1,35 @@
+# EKS
+
+EKS is the chosen implementation of Kubernetes: AWS manages the master nodes but not the worker nodes that run containers; those should be run in an autoscaling group that uses a standard AMI.
+
+A project can be configured to create a cluster with the `eks` configuration in `elife.yaml`.
+
+## User access
+
+### Prerequisites
+
+- Install [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- Install [`aws-iam-authenticator`](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html)
+
+### Configure kubectl
+
+Make sure your AWS user is _either_:
+
+- added to the `KubernetesAdministrators` group
+- allowed to execute `eks:describeCluster`, and `sts:assumeRole` on `arn:aws:iam::512686554592:role/kubernetes-aws--test--AmazonEKSUserRole`
+
+where `kubernetes-aws--test` is the name of the stack containing the cluster.
+
+Execute:
+
+```
+$ aws eks update-kubeconfig --name=kubernetes-aws--test --role-arn arn:aws:iam::512686554592:role/kubernetes-aws--test--AmazonEKSUserRole
```

Comment:
Had to upgrade `awscli` (I was on `1.14.33`, now `1.16.134`).

Now getting `You must specify a region. You can also configure your region by running "aws configure".`.

Reply:
The region we use for almost everything is `us-east-1`, it would be in `~/.aws/config` (or can be added through `aws configure`). Could add `AWS_DEFAULT_REGION=us-east-1`...

Reply:
Okay, that works.